envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -358,16 +361,28 @@ message NotHealthCheckFilter {
 message TraceableFilter {
 }
 
-// Filters for random sampling of requests. Sampling pivots on the header
-// :ref:`x-request-id<config_http_conn_man_headers_x-request-id>` being present. If
-// :ref:`x-request-id<config_http_conn_man_headers_x-request-id>` is present, the filter will
-// consistently sample across multiple hosts based on the runtime key value and the value extracted
-// from :ref:`x-request-id<config_http_conn_man_headers_x-request-id>`. If it is missing, the
-// filter will randomly sample based on the runtime key value.
+// Filters for random sampling of requests.
 message RuntimeFilter {
-  // Runtime key to get the percentage of requests to be sampled. This runtime
-  // control is specified in the range 0-100 and defaults to 0.
+  // Runtime key to get an optional overridden numerator for use in the *percent_sampled* field.
+  // If found in runtime, this value will replace the default numerator.
   string runtime_key = 1 [(validate.rules).string.min_bytes = 1];
+
+  // The default sampling percentage. If not specified, defaults to 0% with denominator of 100.
```

Comment:
Just defaults to 0% would be fine.

Reply:
I think specifying the default denominator is important in the case of runtime override?